lib/install/management/delete.go

```diff
@@ -82,6 +82,13 @@ func (d *Dispatcher) DeleteVCH(conf *config.VirtualContainerHostConfigSpec, cont
 		}
 	}
 
+	element, err := d.session.Finder.Element(d.op, vmm.Reference())
```

Comment:
Will need some eyes on this. Before I did a new Element lookup the inventory path was being incorrectly filled. I can likely just refactor the other call. I did this to get around the InventoryPath mysteriously being `""`.

Reply:
Moved this in to the findApplianceByID function. Should fill the inventory Path correctly for all callers now.